client/src/components/common/ProductItem.tsx

```diff
@@ -126,10 +135,26 @@ const RegionAndDate = styled.p`
 const Price = styled.p`
   font-size: 0.875rem;
 `;
-
+const Status = styled.div<{ status: string }>`
```

Comment:
status 타입이 enum이 되면 좋을 것 같아요. `client/src/types/index.ts`의 status 타입도 ProductStatus로 같이 바꿔주면 될 것 같습니다!

```suggestion
const Status = styled.div<{ status: ProductStatus }>`
```

Reply:
네 이 부분 말고도 빠진부분이 많이 있었네요 수정해뒀습니다!!